app/src/api/jobs.js

```diff
@@ -33,7 +33,7 @@ export const getJobById = async (jobId) => {
   try {
     const res = await axios.get(`${JOB_ENDPOINT}/${jobId}`);
     if (res) {
-      return transformOneJob(res.data);
+      return tsfmOneJobFromSnakeToCamel(res.data);
```

Comment:
unofficial short-forms like this may be a bit confusing, perhaps can change to "transformJobSnakeToCamel" if you'd want, other than that gd to go!